docs/azure_arc_jumpstart/azure_arc_data/gke/gke_dc_vanilla_terraform/_index.md

```diff
@@ -95,51 +88,49 @@ By the end of this scenario, you will have a GKE cluster deployed with an Azure
     }
     ```
 
-    > **NOTE: If you create multiple subsequent role assignments on the same service principal, your client secret (password) will be destroyed and recreated each time. Therefore, make sure you grab the correct password**.
-
     > **NOTE: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/azure/role-based-access-control/best-practices)**
 
 * Create a new GCP Project, IAM Role & Service Account. In order to deploy resources in GCP, we will create a new GCP Project as well as a service account to allow Terraform to authenticate against GCP APIs and run the plan to deploy resources.
 
 * Browse to <https://console.cloud.google.com/> and login with your Google Cloud account. Once logged in, [create a new project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) named "Azure Arc Demo". After creating it, be sure to copy down the project id as it is usually different then the project name.
 
-  ![GCP new project](./01.png)
+  ![GCP new project](./05.png)
 
-  ![GCP new project](./02.png)
+  ![GCP new project](./06.png)
 
-  ![GCP new project](./03.png)
+  ![GCP new project](./07.png)
 
 * Enable the Compute Engine API for the project, create a project Owner service account credentials and download the private key JSON file and copy the file to the directory where Terraform files are located. Change the JSON file name (for example *account.json*). The Terraform plan will be using the credentials stored in this file to authenticate against your GCP project.
 
-  ![Enable Compute Engine API](./04.png)
+  ![Enable Compute Engine API](./08.png)
 
-  ![Enable Compute Engine API](./05.png)
+  ![Enable Compute Engine API](./09.png)
 
-  ![Add credentials](./06.png)
+  ![Add credentials](./10.png)
 
-  ![Add credentials](./08.png)
+  ![Add credentials](./11.png)
```

Comment:
Add some rectangles to point to where the user needs to click and text instructions between screenshots to provide context on what needs to be done